submarine-workbench/workbench-server/src/main/resources/org/apache/submarine/database/mappers/TeamMemberMapper.xml

```diff
@@ -123,12 +118,7 @@
       <if test="inviter != null">
         #{inviter,jdbcType=INTEGER},
       </if>
-      <if test="createBy != null">
-        #{createBy,jdbcType=VARCHAR},
-      </if>
-      <if test="createTime != null">
-        #{createTime,jdbcType=TIMESTAMP},
-      </if>
+      now(), now(),
```

Comment:
`create_by`  = `now()` ??

Reply:
This is my careless mistake. I should set another update_time = now().